sdk/go/common/workspace/project.go

```diff
@@ -109,7 +109,9 @@ type Project struct {
 	// Options is an optional set of project options
 	Options *ProjectOptions `json:"options,omitempty" yaml:"options,omitempty"`
 
-	Providers []PluginOptions `json:"providers,omitempty" yaml:"providers,omitempty"`
+	Providers       []PluginOptions `json:"providers,omitempty" yaml:"providers,omitempty"`
+	LanguagePlugins []PluginOptions `json:"languagePlugins,omitempty" yaml:"languagePlugins,omitempty"`
```

Comment:
I think we should be consistent, and either name all of these Plugins, or just call this languages. Probably the former, these are relatively advanced use fields, it's ok for them to have longer names.